bindings/cs/Scripts/build.cake

```diff
@@ -57,16 +59,9 @@ Task("Prepare")
         UncompressToTheCurrentDirectory(nativeLibrariesArchive);
 
         Information("Cloning the libmongocrypt repo..");
-        GitClone(
-            fork, 
-            csharpBindingsDirectory, 
-            new GitCloneSettings
-            {
-                BranchName = branch,
-                Checkout = true,
-                IsBare = false,
-                RecurseSubmodules = true
-            });
+        EnsureDirectoryExists(csharpBindingsDirectory);
+        Git(csharpBindingsDirectory, $"clone {fork} -b {branch} .");
```

Comment:
Replaced GitClone with Git since GitClone seems to have some native library issues on Mac